### PR TITLE
Add underline on sign in link.

### PR DESCRIPTION
**Pull request by waheedahmed**
Also removed custom PX size classes and used bootstrap classes.

VAN-359

### CODE REVIEW

File: src/register/RegistrationPage.jsx [hunk 1]
```diff
@@ -653,12 +653,16 @@ class RegistrationPage extends React.Component {
                   referrer={REGISTER_PAGE}
                 />
               )}
-              <div className="text-left">
-                <span className="mr-1">{intl.formatMessage(messages['already.have.an.edx.account'])}</span>
-                <a href={LOGIN_PAGE} onClick={this.handleLoginLinkClickEvent}>{intl.formatMessage(messages['sign.in.hyperlink'])}</a>
+              <div className="">
```
Comment (zainab-amir) [inline src/register/RegistrationPage.jsx:656]: kindly remove empty className

File: src/register/RegistrationPage.jsx [hunk 2]
```diff
@@ -653,12 +653,16 @@ class RegistrationPage extends React.Component {
...
+                <p>
```
Comment (zainab-amir) [inline src/register/RegistrationPage.jsx:657]: do we need the <div> tag above? or just <p> is enough

Comment (waheedahmed) [reply src/register/RegistrationPage.jsx:657]: I guess no, let me remove it.

**Comment by waheedahmed** [comment]
@zainab-amir Addressed your comments, kindly take another look.

**Review by zainab-amir — APPROVED** [review]

**Pull request merged**